tests/test_all.py

```diff
@@ -43,6 +43,134 @@
 from nertokenizer import *  # noqa
 from settings import *  # noqa
 
+def checking(text, real):
+    resp = check_grammar(text)
+    i = list([ i["i"] for i in resp[0][0][0]['tokens'] ])
```

Comment:
`list()` is unnecessary around a list comprehension

Reply:
Done